Automattic/wp-calypso
# Update wordpress/editor store calls to import approach

## Proposed Changes
**Note: typecheck is currently failing, but the relevant package will be removed soon. (#75269)**

Related to #74473. The benefits to importing stores directly are:

1. Explicit dependencies, not implicit dependencies.
2. Better types (where applicable).

So this makes sure we reference the 'core/editor` store via an import instead of a string.

The main motivation for this PR was to fix some deprecation warnings on trunk. (Some selectors were moved to core/block-editor instead, so those were also updated.)

## Testing Instructions
Essentially, smoke test as many ETK features as you can, after sandboxing a test site and syncing ETK to your sandbox. This is pretty straightforward, so you can also verify I didn't make any typos :P.
